src/hooks/useWelcomeScreen.ts

```diff
@@ -1,25 +1,42 @@
-import { useEffect, useState } from 'react';
-import { useCookies } from 'react-cookie';
 import { useSettingsStore } from '@/stores/settings';
+import { useEffect, useMemo, useState } from 'react';
+import { useCookies } from 'react-cookie';
 import { useMultisig } from './useMultisig';
 
 interface useWelcomeScreenProps {
   welcomeScreenClosed: boolean | undefined;
   setWelcomeScreenClosed: (closed: boolean) => void;
+  enabled: boolean;
 }
 
 export const useWelcomeScreen = (
   initialState?: boolean,
+  activeTheme?: string,
 ): useWelcomeScreenProps => {
-  const [, setState] = useState(initialState);
-  const [cookie, setCookie] = useCookies(['welcomeScreenClosed']);
+  const [state, setState] = useState(initialState);
+  const [cookie, setCookie] = useCookies(['welcomeScreenClosed', 'theme']);
   const { isMultisigSigner } = useMultisig();
 
   const [, sessionSetWelcomeScreenClosed] = useSettingsStore((state) => [
     state.welcomeScreenClosed,
     state.setWelcomeScreenClosed,
   ]);
 
+  const enabled = useMemo(
+    // check if theme is any of jumper-themes or undefined
+    () =>
+      activeTheme === 'default' ||
+      activeTheme === 'light' ||
+      activeTheme === 'dark' ||
+      activeTheme === 'system' ||
+      cookie.theme === 'default' ||
+      cookie.theme === 'light' ||
+      cookie.theme === 'dark' ||
+      cookie.theme === 'system' ||
+      !cookie.theme,
```

Comment:
Would be better to simplify it

```suggestion
() => {
  const validThemes = ['default', 'light', 'dark', 'system'];
  return validThemes.includes(activeTheme) || validThemes.includes(cookie.theme) || !cookie.theme;
},
```